ConfigurationSystem/Agent/VOMS2CSAgent.py

```diff
@@ -73,11 +74,13 @@ def execute(self):
       autoAddUsers = getVOOption(vo, "AutoAddUsers", self.autoAddUsers)
       autoModifyUsers = getVOOption(vo, "AutoModifyUsers", self.autoModifyUsers)
       autoDeleteUsers = getVOOption(vo, "AutoDeleteUsers", self.autoDeleteUsers)
+      autoLiftSuspendedStatus = getVOOption(vo, "autoLiftSuspendedStatus", self.autoLiftSuspendedStatus)
```

Comment:
This should also go to the ConfigTemplate no ?

Reply:
And should start with a capital letter

Reply:
True